### ISSUE TITLE
First compile in Visual Studio

### ISSUE THREAD

**Issue by hdba** [eyewriter/eyewriter#22]
I am new in these project, i have windows 7 and use visual studio 2012, and i have a problem in the moment of compile the "project.pbxproj" and i get the screen "to attach to the process".

Already have the ps3 eye camera and his software (CL-Eye)

**Comment by hdba** [comment]
@ofZach @ofTheo @eyewriter 

I download the codeblocks and its the same problem in the moment to compile-run
what is the problem? or in the step 3C of instructables, mention that be sure to compile to the current operation system and in my way is win7 but i opened in Visual studio and codebocks

**Comment by ofZach** [comment]
are you building the 1.0 version of the eyewriter?

**Comment by hdba** [comment]
no, the 2.0 version

**Comment by hdba** [comment]
in the project "project.pbxproj" that is in the download flies, i change the settings of MAC system (osx) to windows 7 system ??

**Comment by ofZach** [comment]
project.pbxproj is an OSX / xcode project.   I think you need to install code blocks, but it looks like the CB project might be quite old.  Let me take a look at it.

**Comment by hdba** [comment]
i actually instal de codeblocks but in the 10.05 version, change to the newest version?

**Comment by ofZach** [comment]
you might try this branch of EyeWriter https://github.com/jasonwebb/eyewriter which has more up to date codeblocks files (I think it will compile against OF 0.7.4, but it might be more up to date then that).   I'll take a look later this week, I'm not in front of a windows machine until I'm back in the office mid week.

**Comment by hdba** [comment]
okei, let me check if works, thks i really appreciate your time to helping me and for solve future problems like this

**Comment by leyham** [comment]
Is this working in windows 10? we're supposed to run and build eyewritercode but unfortunately it failed 🙁

**Comment by ofZach** [comment]
sorry this project is almost 8 years old and is unsupported

**Comment by maryam-aldarwish** [comment]
so it not working now? :(

**Comment by leyham** [comment]
yeah sir a bit older,but there are some who work on it and still working a few years ago.

**Comment by ofZach** [comment]
ok !  we are happy to take any pull requests for the project / updates to the code but we sadly can't offer support since we don't have the hardware to test on.

**Comment by leyham** [comment]
i have a question sir. yesterday i build and run the program in the codeblocks 12.4 but there's an error says fatal error glew.h is not in a filebor directory but when i look for it manually i found it but still the cb doesn't read the file is it a problem of a cb? because probably it's not in the code. how can we fix it sir? if i fix one fatal error there's another fatal error when i build and run it. cqn u give me an advice sir?

**Comment by ofZach** [comment]
so sorry but as I've said twice now **we can't offer support**  -- I don't have any answer to fix your cb issues.  We only use visual studio and xcode now with newer openframeworks.

**Comment by ofZach** [comment]
also you might look at some of the other branches of the code and see if someone has done work modernizing the project -- 

<img width="1072" alt="screen shot 2017-11-15 at 4 11 19 am" src="https://user-images.githubusercontent.com/142897/32827718-1bc41524-c9bb-11e7-9643-94bbba7e0e8d.png">

**Comment by Shyckh** [comment]
please provide your email addressi will send you some files may be this will help you    On Wednesday, November 15, 2017, 2:12:19 PM GMT+5, ofZach <notifications@github.com> wrote:  
 
 
also you might look at some of the other branches of the code and see if someone has done work modernizing the project --
